dbmodel/mysql_db_setup_lsdserver.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `license_status` (
+    `id` int(11) PRIMARY KEY AUTO_INCREMENT,
```

Comment:
@llemeurfr I had to add AUTO_INCREMENT here and on event.id. As far as I can see, the id is not set programatically when creating these. Am I just missing something, or is this a bug?

Reply:
You're right, SQLite applies an implicit AUTO_INCREMENT to an INTEGER PRIMARY KEY column (which is used as an an alias for the native "rowid" column), but this is not the case for MySQL.
In this database, the `content` and `license` tables have an `id` column which is in fact a uuid that must be explicitly set. All other tables have an auto incremented `id` column. 

I'll therefore accept this PR but will correct the creation of the other tables where this auto-increment must be set.